src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -43,9 +50,9 @@ def __init__(self):
     for extension in extensions_to_strip:
       app_name = utils.strip_from_right(app_name, extension)
 
-    self.trials = {}
     for trial in data_types.Trial.query(data_types.Trial.app_name == app_name):
-      self.trials[trial.app_args] = trial.probability
+      self.trials[trial.app_args] = AppArgs(trial.probability,
+                                            [trial.contradicts])
```

Comment:
Where did the split go? We'd still want to support multiple possible contradictions that could be listed in DB by space separation... that's also what the comment at the data type says. split on empty string returns an empty list, which would fit.

Reply:
(See comment below)

How about we make this a repeated property so it's automatically a list (and stored as such) instead?

Reply:
done